etc/base.yaml

```diff
@@ -178,17 +178,40 @@ postgresql:
         - name: POSTGRES_MULTIPLE_DATABASES
           value: restsourceauthorizer,appconfig,kratos # ... and these databases will be created
 
+cloudnative_postgresql:
+  _install: true
+  _chart_version: 0.1.0
+  _extra_timeout: 0
```

Comment:
It's good to allow configuration of the storage here since it is something that will be changed more often.

Reply:
Allowing configuration of cluster size would also be good here

Reply:
I added these config options as was already present for the timescaledb instances